docs/conf.py

```diff
@@ -20,6 +20,9 @@
 from django.core.urlresolvers import get_resolver
 from django.utils.html import strip_tags
 
+#Py3 compatible, TODO: rebuild config
+from builtins import str as unicode
```

Comment:
Ej…Widzę.

Reply:
Uznałem to za mniej inwazyjne niż 'import str as text' i zamianę 2 wystąpień unicode na text niżej.
Natomiast jak wspomniałem, proponowałbym odpalić jeszcze raz na świeżo ten cały sphinx-quickstart, tylko nie znam się zbytnio na sphinxie a dziś jest umówiony upgrade